docs/feature-store/sources-targets.md

```diff
@@ -22,6 +19,120 @@
+```
+`SQLSource` can be used for both batch ingestion and real time ingestion. It supports storey but does not support Spark. To configure 
+either, pass the `db_uri` or overwrite the `MLRUN_SQL__URL` env var, in this format:<br> 
+`mysql+pymysql://<username>:<password>@<host>:<port>/<db_name>`, for example:
+
+```
+source = SQLSource(table_name='my_table', 
+                     db_path="mysql+pymysql://abc:abc@localhost:3306/my_db", 
+                     key_field='key',
+                     parse_dates=['timestamp'])
+ 
+ feature_set = fs.FeatureSet("my_fs", entities=[fs.Entity('key')],)
+ feature_set.set_targets([])
+ df = fs.ingest(feature_set, source=source)
+```
+
+## Apache Kafka data source
+
+Example:
+
+```
+from mlrun.datastore.sources import KafkaSource
+
+
+with open('/v3io/bigdata/name.crt') as x: 
+    caCert = x.read()  
+caCert
+
+kafka_source = KafkaSource(
+            brokers=['default-tenant.app.vmdev76.lab.iguazeng.com:9092'],
+            topics="stocks-topic",
+            initial_offset="earliest",
+            group="my_group",
+        )
+        
+run_config = fstore.RunConfig(local=False).apply(mlrun.auto_mount())
+
+stocks_set_endpoint = stocks_set.deploy_ingestion_service(source=kafka_source,run_config=run_config)
+```        
+       
+
+## Confluent Kafka data source
+
+```{admonition} Note
+Tech Preview 
+```
+
+Example:
+
+```
+from mlrun.datastore.sources import KafkaSource
+
+
+with open('/v3io/bigdata/name.crt') as x: 
+    caCert = x.read()  
+caCert
+
+
+kafka_source = KafkaSource(
+        brokers=['server-1:9092', 
+        'server-2:9092', 
+        'server-3:9092', 
+        'server-4:9092', 
+        'server-5:9092'],
+        topics=["topic-name"],
+        initial_offset="earliest",
+        group="test",
+        attributes={"sasl" : {
+                      "enable": True,
+                      "password" : "pword",
+                      "user" : "user",
+                      "handshake" : True,
+                      "mechanism" : "SCRAM-SHA-256"},
+                    "tls" : {
+                      "enable": True,
+                      "insecureSkipVerify" : False
+                    },            
+                   "caCert" : caCert}
+    )
+    
+run_config = fstore.RunConfig(local=False).apply(mlrun.auto_mount())
+
+stocks_set_endpoint = stocks_set.deploy_ingestion_service(source=kafka_source,run_config=run_config)
+```
+
+
+
+
+
+
+
 # Targets
 | Class name                                                                                                    | Description                                                                                        | storey | spark | pandas |
 | --------------------------------------------------                                                            | -------------------------------------------------------                                            | ---    | ---   | ---    |
```

Comment:
1. there are couple of errors here in comparison to my comment in https://jira.iguazeng.com/browse/ML-5680?focusedCommentId=124167&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-124167 (ex in kafkaTarget, , SqlTarget, ParquetTarget) - please recheck
2. Instead of links to storey, I would link to the mlrun section - down below)
3. Suggest to sort by offline/online (targets) and batch/events (sources) and make the detailed description later on follow this order

Reply:
1. done
2. done for existing sections. Need to add more
3. grouped by offline/online (targets) and batch/events (sources)